### DISCUSSION TITLE
VSCode Plugin: Are the editor settings still accurate?

### DISCUSSION THREAD

**Discussion by ghost** [biomejs/biome#1813] [Q&A] [answered]
Currently, the docs state that the editor settings should look like this:

```json
{
  "editor.codeActionsOnSave": {
    "quickfix.biome": true,
    "source.organizeImports.biome": true,
  }
}
```

However, VSCode repeatedly attempts to modify them to `explicit`. Is this setting accurate?

**Comment by nhedger** [comment, marked as answer]
Yes, the docs are outdated. VS Code automatically migrates the setting. You’re welcome to send a PR if you want, otherwise I’ll fix it when I have a minute.

**Reply by ghost** [reply]
Sure thing - https://github.com/biomejs/biome/pull/1828
